src/appdev/ApplicationRouter.ts

```diff
@@ -0,0 +1,122 @@
+import { NextFunction, Request, Response, Router } from 'express';
+
+/**
+ * RequestType - the HTTP methods supported by ApplicationRouter
+ */
+export type RequestType = 'GET' | 'POST' | 'DELETE';
```

Comment:
Are we supposed to use these in routers?

Reply:
Ya, they're used in the ApplicationRouter's constructor.